articles/aks/wasmcloud.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Use wasmCloud on Azure Kubernetes Service (AKS)
+description: Learn how to deploy and use wasmCloud on an Azure Kubernetes Service (AKS) cluster to deploy distributed Web Assembly applications.
+author: protochron
+ms.topic: article
+ms.date: 11/07/2024
+---
+
+# Use wasmCloud on Azure Kubernetes Service (AKS)
+
+[wasmCloud][wasmcloud] is a CNCF project designed to fast-track the development, deployment and orchestration of WebAssembly components. This document details instructions on how to deploy wasmCloud on an Azure Kubernetes Service (AKS) cluster.
+
```

Comment:
Describe what are the benefits of running WebAssembly workloads compare with container workloads